### ISSUE TITLE
Create a demo TODO APP with React / NextJS / Svelte / Vue

### ISSUE THREAD

**Issue by brunobuddy** [mnfst/llm-gateway#206]
We built a [demo repo](https://github.com/mnfst/frontend-examples) to showcase the integration of Manifest with different frontend frameworks.

The demo is based on the popular [TodoMVC website](https://todomvc.com/) that compares the differents frameworks building the same **todo app**. In our use case we added a small Manifest backend with a db for persistant data:

```yaml
name: My TODO App ✅
entities:
  Todo:
    seedCount: 10
    properties:
      - title
      - { name: completed, type: boolean }
```

We need now to create frontend examples based on the TodoMVC ones:
- [ ] React
- [ ] NextJS
- [ ] Svelte
- [ ] Vue
- [ ] Other frontend ?

## How to submit a demo

- Clone the [mnfst/frontend-examples](https://github.com/mnfst/frontend-examples) repository
- Install and checkout the angular implementation for inspiration
- Go to [TodoMVC Github](https://github.com/tastejs/todomvc/tree/master/examples) and grab the framework that you are going to integrate
- Copy it on the `/examples` folder and connect it to the backend
- Make sure that there is an `npm run dev` script in the package.json
- Make your PR in the  [mnfst/frontend-examples](https://github.com/mnfst/frontend-examples) mentionning this issue

**Comment by ChinoUkaegbu** [comment]
Hi, I'd like to work on the React example!

**Comment by dawipayandas** [comment]
I'd like to work under this issue

**Comment by SebConejo** [comment]
Hi @ChinoUkaegbu and @dawipayandas . Which frontend do you want to work with? 
@brunobuddy FYI

**Comment by ChinoUkaegbu** [comment]
I can work on React.

**Comment by ChinoUkaegbu** [comment]
Just wondering what version of npm I should use for installing manifest. I'm getting a few errors when using Node.js v20.10.0 and I suspect it's as a result of that.

**Comment by brunobuddy** [comment]
@ChinoUkaegbu can you send me a screenshot of your errors ?

**Comment by ChinoUkaegbu** [comment]
![Screenshot (430)](https://github.com/user-attachments/assets/0246acfc-3985-4a63-b51f-8934a4cfa39f)
![Screenshot (431)](https://github.com/user-attachments/assets/1cafa34e-4066-4c78-8679-3341c613e0e2)

**Comment by brunobuddy** [comment]
@ChinoUkaegbu it is not tested on Windows, sorry.... Only Mac and Linux for now

**Comment by LLxD** [comment]
Hey @brunobuddy! I'd love to tackle the Svelte one ;). Is it still available?

**Comment by brunobuddy** [comment]
@ChinoUkaegbu I just released a version that fixes that issue on Windows !

**Comment by brunobuddy** [comment]
@LLxD yes !

**Comment by ChinoUkaegbu** [comment]
Thanks! I'll continue working on the React example.

**Comment by ChinoUkaegbu** [comment]
Got this error when trying to run the angular example :( Manifest installed without errors but running `npm run dev` is resulting in this:
![Screenshot (499)](https://github.com/user-attachments/assets/67561622-8478-49be-8c29-4127e0fad01d)
![Screenshot (500)](https://github.com/user-attachments/assets/c881bb68-afad-407a-a877-35c751d1054c)

**Comment by ChinoUkaegbu** [comment]
Good news and bad news:

I got the manifest backend to work. Had to change the nodemonPath in `watch.js` to end with `nodemon.cmd` instead of `nodemon` since that's the Windows specific executable for nodemon. Shown below:
![image](https://github.com/user-attachments/assets/1340364b-4cb1-486d-9f91-eb9a81ed7720)
I'm not sure how to modify the script to handle cross-platform compatibility? I think npx can handle that?

I'll work on the Angular error next but it still persisted.

**Comment by brunobuddy** [comment]
@ChinoUkaegbu sorry I did not explain to you that you had to get the latest version of the "manifest" package. I updated it on the "main" branch you can pull it and it should fix the issue

**Comment by ChinoUkaegbu** [comment]
I believe I was working off the latest version. I pulled the changes before starting.

The errors I got two weeks ago were with adding manifest using `npx add manifest` but they were resolved with the latest update! Now the errors from two days ago were with `npm run manifest` which was because of the `nodemon` executable not being compatible with windows but that was resolved like I mentioned above when I used `nodemon.cmd` instead.

**Comment by LLxD** [comment]
Hey @brunobuddy! Added a PR for the svelte example. Could you give it a check pls? ;)

**Comment by BatatiDE** [comment]
.take

**Comment by brunobuddy** [comment]
@LLxD yes I am going to have a look !

**Comment by LLxD** [comment]
Awesome @brunobuddy! Feel free to reach me in case any questions appear ;)

**Comment by Ron1005** [comment]
Hello @brunobuddy I have implemented Vue and created a pull request, please check and let me know if any changes are required. Thank You !!!

**Comment by brunobuddy** [comment]
@Ron1005 amazing, I will have a look next week !

**Comment by Ron1005** [comment]
@ChinoUkaegbu are you working on the react example ??
@brunobuddy do you mind if I work on the react example if @ChinoUkaegbu is not ??

**Comment by ChinoUkaegbu** [comment]
Hi, I linked a PR for the React example but it hasn't been reviewed yet I believe

**Comment by Ron1005** [comment]
No worries

**Comment by brunobuddy** [comment]
hello @ChinoUkaegbu your PR is sill in draft so I did not review it yet. If you think it's ready you can publish it !

**Comment by brunobuddy** [comment]
@Ron1005 if you are up to you also can do the vanillaJS one

**Comment by ChinoUkaegbu** [comment]
Oh my apologies, I marked it as a draft because I left some comments earlier but I've marked it as ready for review now!

**Comment by Ron1005** [comment]
> @Ron1005 if you are up to you also can do the vanillaJS one

Sure I will give it a try

**Comment by Ron1005** [comment]
@brunobuddy I have opened another pull request to push changes for javascript-es6
